addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/internal/EncryptionHelper.java

```diff
@@ -36,13 +36,13 @@
     // AES‐CBC 128 initial vector, taken from protocol description
     protected static final byte[] IV = parseHexBinary("17996D093D28DDB3BA695A2E6F58562E");
 
-    private static final Logger logger = LoggerFactory.getLogger(EncryptionHelper.class);
```

Comment:
For static helper classes the static loggers are okay, you did not have to change it if you did not want to 😄